src/lib/routing/index.ts

```diff
@@ -0,0 +1,27 @@
+import type { SiteLocale } from '@lib/types/datocms';
+import { getLocale } from '@lib/i18n';
+
+import { getPagePath, type PageOrHomeRoute } from './page';
```

Comment:
Up until now we've only use `import type { ... }` (instead of `import { type ... }` in this project. I thought that was needed to make the distinction between types and runtime code, but I see both are a solution to that. Do we want to stick to a specific pattern, or just mix and match? As in, probably write imports the way to produces the least amount of `import` statements :)

Reply:
I would say, when you're only importing types, add it before the braces, when importing type and code, place it inside the braces. But I can live with separating type imports from code imports altogether. But then the question is, first all type imports, then all code, or (keep it) sorted by path and alternate between type and code?

Again my (not very deep-seated) preference is:
* sort/group by import path (dependencies first, aliased source files second, relative imports last)
* inside one `import {...} from ''` import, code first (alphabetically), types second (alphabetically)
* no enforcement of this with tooling, if the order is wrong no biggie.